epub33/core/index.html

```diff
@@ -1001,6 +1002,30 @@ <h4>Resource Locations</h4>
 					</div>
 				</section>
 
+				<section id="sec-data-urls">
+					<h4>Data URLs</h4>
+
+					<p>The <a href="https://tools.ietf.org/html/rfc2397"><code>data:</code> URL scheme</a> [[!RFC2397]]
+						is used to encode resources directly into a URL string. The advantage of this scheme is that it
+						allows a resource to be embedded within another, avoiding the need for an external file.</p>
+
+					<p><a>EPUB Creators</a> MAY use data URLs in EPUB Publications provided their use does not result in
+						a <a>Top-Level Content Document</a> or open a new browser instance (e.g., they cannot be used in
+						manifest <code>item</code> elements that are referenced from the <a>spine</a>, in [[HTML]] or
+						[[SVG]] <code>a</code> elements (except when inside an <code>iframe</code>), or in calls to
+						[[ECMASCRIPT]] <code>window.open</code> or <code>document.open</code>).</p>
```

Comment:
This list is exhaustive enough that it may need clarification that it is not exhaustive (in addition the "e.g." at the start). Actually might be better to introduce a new sentence here, since you have a parenthetical in the parenthetical.

Reply:
It's nearly exhaustive in that I can only see that the area element is missing from it. I'll add that case in and add a note that it's still subject to change as html, svg, etc. evolve.